README.md

```diff
@@ -4,42 +4,41 @@
 </p>
 
 <!-- Uncomment these lines after releasing the package to PyPI for version and downloads badges -->
-<!--[![PyPI Shield](https://img.shields.io/pypi/v/pygridsim.svg)](https://pypi.python.org/pypi/pygridsim)-->
-<!--[![Downloads](https://pepy.tech/badge/pygridsim)](https://pepy.tech/project/pygridsim)-->
+[![Development Status](https://img.shields.io/badge/Development%20Status-2%20--%20Pre--Alpha-yellow)](https://pypi.org/search/?c=Development+Status+%3A%3A+2+-+Pre-Alpha)
+[![PyPI Shield](https://img.shields.io/pypi/v/pygridsim.svg)](https://pypi.python.org/pypi/pygridsim)
+[![Downloads](https://pepy.tech/badge/pygridsim)](https://pepy.tech/project/pygridsim)
+[![Run Tests](https://github.com/DAI-Lab/PyGridSim/actions/workflows/tests.yml/badge.svg)](https://github.com/DAI-Lab/PyGridSim/actions/workflows/tests.yml)
 [![Github Actions Shield](https://img.shields.io/github/workflow/status/amzhao/PyGridSim/Run%20Tests)](https://github.com/amzhao/PyGridSim/actions)
 [![Coverage Status](https://codecov.io/gh/amzhao/PyGridSim/branch/master/graph/badge.svg)](https://codecov.io/gh/amzhao/PyGridSim)
 
 
 
 # PyGridSim
 
-Package to simulate OpenDSS circuits on Python.
+PyGridSim is a package for simulating OpenDSS circuits. PyGridSim uses a functional interface to allow users to efficiently generate circuits of various scopes.
 
-NOTE: README comes from dai cookie cutter, will be updated
-NOTE: will be moved to Dai lab repository
-
-- Documentation: https://amzhao.github.io/PyGridSim
-- Homepage: https://github.com/amzhao/PyGridSim
+- Documentation: https://dtail.gitbook.io/pygridsim
+- Homepage: https://dai.lids.mit.edu/
```

Comment:
homepage is the package homepage: https://github.com/DAI-Lab/PyGridSim/ 
not the lab's website